target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -157,11 +157,8 @@ ramips_setup_interfaces()
 	tplink,mr600-v2-eu)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
 		;;
-	ubnt,edgerouter-x)
-		ucidef_set_interfaces_lan_wan "eth1 eth2 eth3 eth4" "eth0"
-		;;
```

Comment:
Why did you remove lines 160 to 162?
Shouldn't it be: ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan" ?